jina/proto/jina.proto

```diff
@@ -285,6 +285,16 @@ message DocumentArrayProto {
     repeated DocumentProto docs = 1; // a list of Documents
 }
 
+/**
+ * Represents an entity (like a Pea)
```

Comment:
```suggestion
 * Represents an entity (like an ExecuorRuntime) 
```
Let's prepare for the separation of concerns, proto does not know about Peas and pods